frontend/packages/helm-plugin/integration-tests/features/helm/actions-on-helm-release.feature

```diff
@@ -0,0 +1,59 @@
+@helm
+Feature: Perform Actions on Helm Releases
+              As a user, I want to perform the actions on the helm releases in topology page
```

Comment:
Indentation

Reply:
It is automatically updated on save, on gherkin-lint able to see this issue